src/app/components/current-user/current-user.component.html

```diff
@@ -1,6 +1,11 @@
 <div title="{{ (profile$ | async)?.id }}">
   <div class="current-user__full-name">{{ (profile$ | async)?.userName }}</div>
-  <div class="current-user__avatar am-avatar" [matMenuTriggerFor]="userMenu">
+  <div
+    role="button"
+    [attr.aria-label]="(profile$ | async)?.initials | translate"
```

Comment:
since its marked as a button, it also must be accessible via keyboard like the native button element. this means it needs also the tabindex="0" added to attribute list

Reply:
Thanks :)